### PR TITLE
add pageModel references endopoint

### PR DESCRIPTION
**Pull request by stefanopuddu**

### CODE REVIEW

File: engine/src/main/java/org/entando/entando/web/pagemodel/PageModelController.java [hunk 1]
```diff
@@ -86,6 +87,13 @@ public void setPageModelValidator(PageModelValidator pageModelValidator) {
         return new ResponseEntity<>(new RestResponse(pageModelDto), HttpStatus.OK);
     }
 
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @RequestMapping(value = "/{code}/references/{manager}", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<?> getGroupReferences(@PathVariable String code, @PathVariable String manager, RestListRequest requestList) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/pagemodel/PageModelController.java]: This looks like it should be getPageReferences

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Pull request merged**